tests/README.md

```diff
@@ -2,8 +2,8 @@
 
 The Rook Test Framework is used to run end to end and integration tests on Rook. The framework depends on a running instance of kubernetes. 
 The framework also provides scripts for starting kubernetes using kubeadm or minikube so users can 
-quickly spin up a kubernetes platform. The Kubeadm-dind.sh script starts a three-node kubernetes cluster (using docker-in-docker) 
-and the minikube.sh starts a single-node kubernetes. The Test framework is designed to install Rook, run tests and uninstall Rook.
+quickly spin up a kubernetes platform. The Kubeadm-dind.sh script starts a three-node kubernetes cluster (using docker-in-docker) or
+kubeadm.sh to start single node kubernetes via kubeadm or minikube.sh  to starts a single-node kubernetes. The Test framework is designed to install Rook, run tests and uninstall Rook.
```

Comment:
Does the test framework use any of these scripts? Or does the framework just use one of them, and the others are provided for developer convenience? 
And a couple wording suggestions:
-  typo: `  to starts`
- comma: `run tests, and uninstall Rook`